test/controllers/visitation.controller.test.js

```diff
@@ -53,9 +70,16 @@ describe('Visitation Controller', () => {
             expect(res.body.success).to.be.eql(true);
             expect(res.body.message).to.be.eql('Visit scheduled successfully');
             expect(res.body).to.have.property('schedule');
-            expect(propId.equals(res.body.schedule.propertyId)).to.be.eql(true);
+            expect(demoPropertyId.equals(res.body.schedule.propertyId)).to.be.eql(true);
```

Comment:
Can we update this to a more readable format, 🙏 

```suggestion
            expect(demoPropertyId.toString()).to.be.eql(res.body.schedule.propertyId.toString());
```

Reply:
you suggested this format a while back tho